src/Tribe/I18n.php

```diff
@@ -73,16 +111,25 @@ public function get_i18n_strings( $strings, $languages, $domains = array(), $def
 	 * Differently from the `get_i18n_strings` method this will not use any domain that's not specified.
 	 *
 	 * @since 5.1.1
+	 * @since TBD   Add support for the $flags argument.
```

Comment:
:no_entry_sign: There must be a single blank line after a tag group

<sub>hash: d9c3b09cbadb0540d8a300f4bcbd41cb</sub>